theme.json

```
@@ -97,6 +97,63 @@
 			],
 			"units": ["%", "px", "em", "rem", "vh", "vw"]
 		},
+		"typography": {
+			"defaultFontSizes": false,
+			"fluid": true,
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Extra Small",
+					"size": "14px",
```

We may want to use `rem` for consistency.

```suggestion
					"size": "0.875rem",
```